docs/config/README.md

```diff
@@ -675,13 +675,22 @@ The `git_branch` module shows the active branch of the repo in your current dire
 
 ### Options
 
-| Variable            | Default         | Description                                                                           |
-| ------------------- | --------------- | ------------------------------------------------------------------------------------- |
-| `symbol`            | `" "`          | The symbol used before the branch name of the repo in your current directory.         |
-| `truncation_length` | `2^63 - 1`      | Truncates a git branch to X graphemes                                                 |
-| `truncation_symbol` | `"…"`           | The symbol used to indicate a branch name was truncated. You can use "" for no symbol |
-| `style`             | `"bold purple"` | The style for the module.                                                             |
-| `disabled`          | `false`         | Disables the `git_branch` module.                                                     |
+| Variable            | Default                          | Description                                                                            |
+| ------------------- | -------------------------------- | -------------------------------------------------------------------------------------- |
+| `format`            | `"on [$symbol$branch]($style) "` | The format for the module.  Use `"$branch"` to refer to the current branch name.       |
+| `symbol`            | `" "`                           | A format string representing the symbol of git branch.                                 |
+| `style`             | `"bold purple"`                  | The style for the module.                                                              |
+| `truncation_length` | `2^63 - 1`                       | Truncates a git branch to X graphemes.                                                 |
+| `truncation_symbol` | `"…"`                            | The symbol used to indicate a branch name was truncated. You can use "" for no symbol. |
```

Comment:
Not related to your change but it would be nice add the following 
```suggestion
| `truncation_symbol` | `"…"`                            | The symbol used to indicate a branch name was truncated. You can use `""` for no symbol. |
```